src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Sve.cs

```diff
@@ -192,5 +192,246 @@ public new abstract class Arm64 : AdvSimd.Arm64
         /// </summary>
         public static unsafe Vector<double> LoadVector(Vector<double> mask, double* address) => LoadVector(mask, address);
 
+
+        ///  LoadVectorByteZeroExtendToInt16 : Load 8-bit data and zero-extend
+
+        /// <summary>
+        /// svint16_t svld1ub_s16(svbool_t pg, const uint8_t *base)
+        ///   LD1B Zresult.H, Pg/Z, [Xarray, Xindex]
+        ///   LD1B Zresult.H, Pg/Z, [Xbase, #0, MUL VL]
```

Comment:
could you please confirm which of the `ld1*` instruction this API maps to? Is it https://docsmirror.github.io/A64/2023-06/ld1b_z_p_ai.html?

Reply:
The disassembly gives `ld1b    { z16.h }, p7/z, [x0]`:

```
Beginning scenario: RunBasicScenario_Load
10:50:41.891 Passed test: _Sve_ro::JIT.HardwareIntrinsics.Arm._Sve.Program.SveLoadVector_ulong()
10:50:41.892 Running test: _Sve_ro::JIT.HardwareIntrinsics.Arm._Sve.Program.SveLoadVectorByteZeroExtendToInt16()
; Assembly listing for method JIT.HardwareIntrinsics.Arm._Sve.LoadUnaryOpTest__SveLoadVectorByteZeroExtendToInt16:RunBasicScenario_Load():this (Tier0)
; Emitting BLENDED_CODE for generic ARM64 - Unix
; Tier0 code
; fp based frame
; partially interruptible
; Final local variable assignments
;
;  V00 this         [V00    ] (  1,  1   )     ref  ->  [fp+0x48]  do-not-enreg[] this class-hnd <JIT.HardwareIntrinsics.Arm._Sve.LoadUnaryOpTest__SveLoadVectorByteZeroExtendToInt16>
;  V01 loc0         [V01    ] (  1,  1   )  simd16  ->  [fp+0x30]  HFA(simd16)  do-not-enreg[S] must-init <System.Numerics.Vector`1[short]>
;  V02 loc1         [V02    ] (  1,  1   )  simd16  ->  [fp+0x20]  HFA(simd16)  do-not-enreg[S] must-init <System.Numerics.Vector`1[short]>
;# V03 OutArgs      [V03    ] (  1,  1   )  struct ( 0) [sp+0x00]  do-not-enreg[XS] addr-exposed "OutgoingArgSpace"
;  V04 tmp1         [V04    ] (  1,  1   )    long  ->  [fp+0x18]  do-not-enreg[] "non-inline candidate call"
;  V05 tmp2         [V05    ] (  1,  1   )    long  ->  [fp+0x10]  do-not-enreg[] "argument with side effect"
;
; Lcl frame size = 64

G_M30813_IG01:  ;; offset=0x0000
            stp     fp, lr, [sp, #-0x50]!
            mov     fp, sp
            str     xzr, [fp, #0x30]	// [V01 loc0]
            str     xzr, [fp, #0x38]	// [V01 loc0+0x08]
            str     xzr, [fp, #0x20]	// [V02 loc1]
            str     xzr, [fp, #0x28]	// [V02 loc1+0x08]
            str     x0, [fp, #0x48]	// [V00 this]
						;; size=28 bbWeight=1 PerfScore 6.50
G_M30813_IG02:  ;; offset=0x001C
            movz    x0, #0xBE18
            movk    x0, #0x8423 LSL #16
            movk    x0, #0xFFFF LSL #32
            movz    x1, #0x7438      // code for TestLibrary.TestFramework:BeginScenario(System.String)
            movk    x1, #0x3AB9 LSL #16
            movk    x1, #0xFFFF LSL #32
            ldr     x1, [x1]
            blr     x1
            ptrue   p7.h
            mov     z16.h, p7/z, #1
            str     q16, [fp, #0x30]	// [V01 loc0]
            ldr     x0, [fp, #0x48]	// [V00 this]
            ldrsb   wzr, [x0]
            ldr     x0, [fp, #0x48]	// [V00 this]
            add     x0, x0, #16
            movz    x1, #0x7DE0      // code for JIT.HardwareIntrinsics.Arm._Sve.LoadUnaryOpTest__SveLoadVectorByteZeroExtendToInt16+DataTable:get_inArray1Ptr():ulong:this
            movk    x1, #0x3AD1 LSL #16
            movk    x1, #0xFFFF LSL #32
            ldr     x1, [x1]
            blr     x1
            ldr     q16, [fp, #0x30]	// [V01 loc0]
            ptrue   p7.h
            cmpne   p7.h, p7/z, z16.h, #0
            ld1b    { z16.h }, p7/z, [x0]
            str     q16, [fp, #0x20]	// [V02 loc1]
            ldr     x0, [fp, #0x48]	// [V00 this]
            ldrsb   wzr, [x0]
            ldr     x0, [fp, #0x48]	// [V00 this]
            add     x0, x0, #16
            movz    x1, #0x7DF8      // code for JIT.HardwareIntrinsics.Arm._Sve.LoadUnaryOpTest__SveLoadVectorByteZeroExtendToInt16+DataTable:get_outArrayPtr():ulong:this
            movk    x1, #0x3AD1 LSL #16
            movk    x1, #0xFFFF LSL #32
            ldr     x1, [x1]
            blr     x1
            ldr     q16, [fp, #0x20]	// [V02 loc1]
            str     q16, [x0]
            ldr     x0, [fp, #0x48]	// [V00 this]
            ldrsb   wzr, [x0]
            ldr     x0, [fp, #0x48]	// [V00 this]
            add     x0, x0, #16
            movz    x1, #0x7DE0      // code for JIT.HardwareIntrinsics.Arm._Sve.LoadUnaryOpTest__SveLoadVectorByteZeroExtendToInt16+DataTable:get_inArray1Ptr():ulong:this
            movk    x1, #0x3AD1 LSL #16
            movk    x1, #0xFFFF LSL #32
            ldr     x1, [x1]
            blr     x1
            str     x0, [fp, #0x18]	// [V04 tmp1]
            ldr     x0, [fp, #0x48]	// [V00 this]
            ldrsb   wzr, [x0]
            ldr     x0, [fp, #0x48]	// [V00 this]
            add     x0, x0, #16
            movz    x1, #0x7DF8      // code for JIT.HardwareIntrinsics.Arm._Sve.LoadUnaryOpTest__SveLoadVectorByteZeroExtendToInt16+DataTable:get_outArrayPtr():ulong:this
            movk    x1, #0x3AD1 LSL #16
            movk    x1, #0xFFFF LSL #32
            ldr     x1, [x1]
            blr     x1
            str     x0, [fp, #0x10]	// [V05 tmp2]
            ldr     x2, [fp, #0x10]	// [V05 tmp2]
            ldr     x1, [fp, #0x18]	// [V04 tmp1]
            ldr     x0, [fp, #0x48]	// [V00 this]
            movz    x3, #0xBE18
            movk    x3, #0x8423 LSL #16
            movk    x3, #0xFFFF LSL #32
            movz    x4, #0x7EE8      // code for JIT.HardwareIntrinsics.Arm._Sve.LoadUnaryOpTest__SveLoadVectorByteZeroExtendToInt16:ValidateResult(ulong,ulong,System.String):this
            movk    x4, #0x3AD1 LSL #16
            movk    x4, #0xFFFF LSL #32
            ldr     x4, [x4]
            blr     x4
						;; size=268 bbWeight=1 PerfScore 98.00
G_M30813_IG03:  ;; offset=0x0128
            ldp     fp, lr, [sp], #0x50
            ret     lr
						;; size=8 bbWeight=1 PerfScore 2.00

; Total bytes of code 304, prolog size 24, PerfScore 106.50, instruction count 76, allocated bytes for code 304 (MethodHash=c5fe87a2) for method JIT.HardwareIntrinsics.Arm._Sve.LoadUnaryOpTest__SveLoadVectorByteZeroExtendToInt16:RunBasicScenario_Load():this (Tier0)
; ============================================================
```

Reply:
> The disassembly gives `ld1b { z16.h }, p7/z, [x0]`:

Just 1 register address. Due to the shortcut in `emitInsSve_R_R_R()`, it drops into `emitIns_R_R_R_I` using an immediate of 0. Which means.....

> I believe it's https://docsmirror.github.io/A64/2023-06/ld1b_z_p_bi.html . @a74nh, could you confirm this JIC?

.....Yes, that one. Contiguous load unsigned bytes to vector (immediate index).

So, we only want the one in the API file:
```
        ///   LD1B Zresult.H, Pg/Z, [Xbase, #0, MUL VL]
```

Reply:
Thanks, that entry from all the API documentation should go away.